### DISCUSSION TITLE
Rename the components folder to ui?

### DISCUSSION THREAD

**Discussion by maeertin** [noaignite/create-ignite-app#75] [Ideas]
Hey team,

What's your opinion on renaming the `components` folder to `ui`. Reason being that everything in react world is basically a component and it can be confusing as to what belongs there. Renaming this to ui or similar might clear up some confusion. What's your thoughts?

@adamsoderstrom @janmysiak @alexanderflink @axeen

**Comment by adamsoderstrom** [comment]
Hey, @maeertin!

See what you're getting at.
I might actually think that the "React" perspective motivates the name `components`, rather than the opposite.

My take on the `components` classification is that entities in this folder should be:
- Isolated
- Simple

Which would make the assumption that every entity is solely one component.

I think that the `ui` doesn't fulfill this distinction as well as the `components` directory does. For example, i would say that the `containers` folder could be subordinate the `ui` folder, because it `containers` is dependent on some "UI" aspects.

What do you say?

**Comment by alexanderflink** [comment]
I'm on the fence about this one! I agree that it is confusing that a "component" means any React component *and* also a "dumb" presentational component. On the other hand, I think a lot of people are used to seeing a "components" folder.

Maybe it's the "containers" folder that should change name? Not sure what though. Maybe "molecules" from atomic design? Or parts? 

Or maybe it is actually enough with the excellent description you wrote in the create-ignite-app readme @maeertin! 😀

**Comment by maeertin** [comment]
Also not sure about this one, let's let it marinate further 😄
